test/micro/org/openjdk/bench/vm/lang/LockUnlock.java

```diff
@@ -312,7 +312,7 @@ private synchronized int fact(int n) {
      * With two threads lockObject1 will be contended so should be
      * inflated.
      */
-    @Threads(2)
+    @Threads(3)
```

Comment:
Please explain this change and update the comment.

Reply:
he reason I increased the number of threads from 2 to 3, was because it enabled me to increase the code coverage, and thereby execute all(?) the corner cases when doing ObjectMonitor locking. I'll update the comment.

Reply:
fixed